src/v/config/configuration.cc

```diff
@@ -98,9 +98,9 @@ configuration::configuration()
       "Lower bound on topic segment.ms: lower values will be clamped to this "
       "value",
       {.needs_restart = needs_restart::no,
-       .example = "60000",
+       .example = "600000",
        .visibility = visibility::tunable},
-      60s)
+      10min)
```

Comment:
I'm wondering if we should also make the `log_segment_ms` time higher too, and allow it to be controlled with something similar to `__REDPANDA_LICENSE_CHECK_INTERVAL_SEC`? At least, there's precedence to test-only env configs, though it's not widely used, and long term may not be what we want (might be worth further discussion).

Generally I'm wondering how we'll explain this in docs: the log_segment_ms range starts at 60s above, which may be where docs gets the values? Should we just get rid of the .min above altogether?

Reply:
ah, i see the problem.

I think the reasoning was for log_segment_ms.min to be the hard limit for the property, while log_segment_ms_min was the "reasonable" limit for normal operations.

also log_segment_ms.min defines the lowest number accepted for the property, which is conceptually different from the actual value computed at runtime, where log_segment_ms_min plays a part.

I could move the .min bound from log_segment_ms to log_segment_ms_min; it should produce the same result and be easier to explain.

Having a low .min bound was done for compatibility reasons, in case people were already setting the value (redpanda accepted log_segment_ms already but did nothing with it)

I don't know how I feel about using an env var for testing purposes. On one hand, we could get rid of log_segment_ms_min and hardcode it to 10min in the documentation (while retaining the "escape hatch" function); on the other hand, @NyaliaLui is working on locked properties that will replace log_segment_ms_min altogether, and this pr was meant to be a small one to remove a foot gun and have something to backport if we feel like it's worth having in 23.1/23.2

Reply:
Yeah, I only bring up changing `log_segment_ms` range because I'm wondering if this page is how docs get populated (https://docs.redpanda.com/current/reference/cluster-properties/#log_segment_ms). OTOH, in that regard, if the idea is that these are already conservative, maybe we shouldn't document the `_min/_max` properties at all